application/actions/meetings_test.go

```diff
@@ -61,3 +62,76 @@ func (as *ActionSuite) Test_MeetingsList() {
 	as.NotContains(body, mtgs[1].Name, "should not have included name of recent meeting")
 
 }
+
+func (as *ActionSuite) Test_meetingsJoin() {
+	f := createFixturesForMeetings(as)
+
+	type testCase struct {
+		name       string
+		inviteCode string
+		location   models.Location
+		meeting    models.Meeting
+		user       models.User
+		wantInvite models.MeetingInvite
+	}
+
+	testCases := []testCase{
```

Comment:
The three test cases all follow the same path in `meetingsJoin` so they're more about testing the logic in the models functions. A test case for a bad/missing meeting might be important, for example. I'm OK with leaving this as it is, though, to get to the point where we can finish the REST conversion sooner.

Reply:
:+1: